Makefile

```diff
@@ -424,17 +428,12 @@ install/yaml/02-open-match-demo.yaml: build/toolchain/bin/helm$(EXE_EXTENSION)
 		--set usingHelmTemplate=true \
 		install/helm/open-match > install/yaml/02-open-match-demo.yaml
 
-install/yaml/03-prometheus-chart.yaml: build/toolchain/bin/helm$(EXE_EXTENSION)
+install/yaml/03-open-match-override-configmap.yaml: build/toolchain/bin/helm$(EXE_EXTENSION)
 	mkdir -p install/yaml/
 	$(HELM) template --name $(OPEN_MATCH_RELEASE_NAME) --namespace $(OPEN_MATCH_KUBERNETES_NAMESPACE) \
-		--set global.image.registry=$(REGISTRY) \
-		--set global.image.tag=$(TAG) \
-		--set open-match-customize.enabled=false \
-		--set open-match-demo.enabled=false \
-		--set open-match-core.enabled=false \
-		--set global.telemetry.prometheus.enabled=true \
-		--set usingHelmTemplate=true \
-		install/helm/open-match > install/yaml/03-prometheus-chart.yaml
+		--set open-match-override.enabled=true \
```

Comment:
Shouldn't core be false too?  Or was that switched to default false?  (If so, this should have been cleaned up separately?)

Reply:
`-x templates/om-configmap-override.yaml` only template this specified file